helpers/lib/common.js

```diff
@@ -0,0 +1,33 @@
+function isObject(val) {
+    return val !== null && typeof val === 'object';
+}
+
+function isOptions(val) {
+    return isObject(val) && val.fn;
+}
+
+function isString(val) {
+    return typeof val === 'string';
+}
+
+function isEmptyObject(val) {
+    return isObject(val) && Object.keys(val).length === 0;
+}
+
+function truthCheck(val) {
```

Comment:
Let's call this `isTruthy` (that's kind of a semi-standard term for this)

Reply:
done.